smoke_tests/README.md

```diff
@@ -0,0 +1,30 @@
+
+
+## Purpose
+To provide a low overhead way to check that BPM is basically operational in a given environment.
```

Comment:
Nit: `s/basically//g` It's either operational or it isn't.

Reply:
I meant that in terms of "the basic functionality is operational", not "the functionality is basically operational".